virus/sedimentum_occultus.md

```diff
@@ -0,0 +1,111 @@
+
+## Abstract
+
+Viruses are found in many natural environments. In particular, the nucleic acids from microorganism samples taken from the environment can be analyzed for the presence of viral genomes or portions of them. A study performed to determine culturing techniques for bacteria sampled from marine sediments in intertidal zones found a possible viral genome within the sample metagenome. This genome was first identified as a possible viral genome because sections of it indicate that it might encode an RNA-dependent RNA polymerase (RdRp), a protein essential to all RNA viruses. However, the virus associated with this genome has not yet been characterized, and many of its attributes are unknown. Here we show that this virus likely belongs to the family *Picornaviridae*. We performed BLAST queries using the ORFs of the virus to find other organisms with similar proteins. Reviewing the results showed that taxonomically similar viruses were found in environments much like the marine sediment where the original virus was found. Our results show several putative features of this virus compared to other viruses in the family *Picornaviridae*. We expect that the topics presented here can be greatly expanded upon to better characterize the virus, as further research is required to determine its host and taxonomic classification.
+
+## Results
+
+### Index Case
+
+*Sedimentum occultus* was originally found in an experiment to determine cultivation procedures for microorganisms that may fail to grow in culture due to their natural growth states, such as remaining dormant [Mu et al. 2018]. The researchers collected three samples of sediment from the intertidal zone of Weihai, China and and extracted DNA and RNA during and after culturing the microbes present in the samples. After removal of rRNA, the remaining RNA was used to synthesize cDNA.
+
+The index run of *S. occultus* was SRR6808584, and its corresponding assembly file contained 1,346,747 nodes in FASTA format. Node 389 was an DNA sequence of length 8.9 kilobases. The RdRp nucleotide sequence of *S. occultus* was present within Node 389. We first used BLAST on the RdRp amino acid sequence to narrow down the virus's potential identity [Altschul et al. 1990]. Many of the BLAST hits with the greatest percent identities showed that the RdRp amino acid sequence was similar to the RdRp amino acid sequences of other unknown viruses in the family *Picornaviridae*. We can moderately conclude from this finding that *S. occultus* belongs to the family *Picornaviridae*. However, since the specific genus and species of the viruses were unknown, it is inconclusive as to whether *S. occultus* has already been discovered and named.
```

Comment:
For all accession codes in the markdown file (SRR, BioProject, GenBank) could you please change them to `code text like this`?